hal/src/timer.rs

```diff
@@ -73,7 +88,6 @@ where
         // How many cycles of the clock need to happen to reach our
```

Comment:
Either move the overflow check & the comment up to the division or the division down here

Reply:
(same for the other file)